packages/validation/src/validators/custom/config.ts

```diff
@@ -1,5 +1,5 @@
 import type {Document, ParsedNode} from 'yaml';
-import {RuleLevel, reportingDescriptorRelationship} from '../../node';
+import {RuleConfigurability, RuleLevel, reportingDescriptorRelationship} from '../../node';
```

Comment:
node.js'; 

iirc, imports must end with .js normally, not sure why this is working as it already was like this 🤔 Maybe because it area only types?

Reply:
Fixed 👍